File: BubbleSort/bubble-sort.py
test = [9,7,6,5,4,3,2,1,0]

def bubbleSort(array):
    n = len(array)
    for j in range(n - 1):
        for i in range(n - 1):
            if array[i] > array[i + 1]:
                aux = array[i]
                array[i] = array[i+1]
                array[i + 1] = aux
    return array

def printA(array):
    for i in array:
        print(i, end=' ')

printA(test)

bubbleSort(test)
print()

printA(test)

File: BubbleSort/bubbleSort.js
let set = [];

 function randonArray(){
     for(let i=0; i < 41; i++){
         set.push(Math.floor(Math.random() * 101))
     }
 }

randonArray()

const bubbleSort = vector => {
    if(vector.length === 0) return "Empty List"

    const n = vector.length;
    for(let j=0; j < n; j++){
        for(let i=0; i < n; i++){
            if(vector[i] > vector[i + 1]){
                let aux = vector[i];
                vector[i] = vector[i + 1];
                vector[i + 1] = aux            }
        }
    }
    return vector;
};

console.log(JSON.stringify(set));

console.log(JSON.stringify(bubbleSort(set)))